x-pack/plugins/endpoint/common/types.ts

```diff
@@ -647,9 +647,16 @@ interface HostPolicyResponseActions {
   [key: string]: HostPolicyResponseActionDetails;
 }
 
+export interface HostPolicyResponseConfiguration {
+  malware: HostPolicyResponseConfigurationStatus;
+  events: HostPolicyResponseConfigurationStatus;
+  logging: HostPolicyResponseConfigurationStatus;
+  streaming: HostPolicyResponseConfigurationStatus;
+}
+
 interface HostPolicyResponseConfigurationStatus {
   status: HostPolicyResponseActionStatus;
-  concerned_actions: Array<keyof HostPolicyResponseActions>;
+  concerned_actions: string[];
```

Comment:
Why the change here?

Reply:
is it to make this more permissive of future actions?

Reply:
i needed this to get a typescript thing to work. otherwise when I tried to use it in a component it complained that the type was either a string or a number, but it would only accept strings.

Reply:
It was already permissive (`HostPolicyResponseActions`) defined open ended keys. :)

Reply:
so for some reason 
![image](https://user-images.githubusercontent.com/56409205/81115123-35da8f80-8ef1-11ea-92f3-c609003332bf.png) this thing was causing a problem. since the key can be a number

Reply:
@parkiino I'm guess it's complaining that it's either a string or number because we're looking at the key of `HostPolicyResponseActions` as the type and implicitly keys can be numbers.  I'd like to understand why, but I would be OK figuring it out in another PR.

Any ideas @paul-tavares ?

Reply:
@parkiino ignore my comment, I was writing it before I saw your response

Reply:
We can work offline on this and look to see what might have been happening. In the data generator, I actually had it returning "unknown" actions and it did not cause an error (that I could see), so I'm interested to look at the usage that was causing the error.

BTW - I think @nnamdifrankie removed the permissive definition in his PR due to the fact that these need to be know upfront for the elasticsarch schema.